test_project/tests/integration_tests/test_admin/test_import/test_admin_actions.py

```diff
@@ -0,0 +1,154 @@
+from django.contrib.auth.models import User
+from django.test.client import Client
+from django.urls import reverse
+
+from rest_framework import status
+
+import pytest
+import pytest_mock
+
+from import_export_extensions.models import ImportJob
+from test_project.fake_app.factories import ArtistImportJobFactory
+
+
+@pytest.mark.django_db(transaction=True)
+def test_cancel_import_admin_action(
+    client: Client,
+    superuser: User,
+    mocker: pytest_mock.MockerFixture,
+):
+    """Test `cancel_import` via admin action."""
+    client.force_login(superuser)
+
+    revoke_mock = mocker.patch("celery.current_app.control.revoke")
+    import_data_mock = mocker.patch(
+        "import_export_extensions.models.ImportJob.import_data",
+    )
+    artist_import_job = ArtistImportJobFactory()
+    artist_import_job.refresh_from_db()
+    artist_import_job.confirm_import()
+
+    response = client.post(
+        reverse("admin:import_export_extensions_importjob_changelist"),
+        data={
+            "action": "cancel_jobs",
+            "_selected_action": [artist_import_job.pk],
+        },
+        follow=True,
+    )
+    artist_import_job.refresh_from_db()
+
+    assert response.status_code == status.HTTP_200_OK
+    assert artist_import_job.import_status == ImportJob.ImportStatus.CANCELLED
+    assert (
+        response.context["messages"]._loaded_data[0].message
+        == f"Import of {artist_import_job} canceled"
+    )
+    import_data_mock.assert_called_once()
+    revoke_mock.assert_called_once_with(
+        artist_import_job.import_task_id,
+        terminate=True,
+    )
+
+
+@pytest.mark.django_db(transaction=True)
+def test_cancel_import_admin_action_with_incorrect_import_job_status(
```

Comment:
Not obvious what statuses are correct and what are incorrect

I think for tests like this it's better to describe choices in docstring and set status explicitly (or provide comments. Now it's default status inside factory)